### ISSUE TITLE
Using Safety seems to disable trap location rendering even for following floors

### ISSUE THREAD

**Issue by KalasB** [carvelli/PalacePal#18]
In the newest version, using a pom of safety once stops the plogon from rendering traps on the next floors.
This behaviour did not occur using previous versions.

Noticed in PotD and EO so far. Have not tried HoH.

**Comment by syldrathecat** [comment]
Oh I am not the only one. I was trying to figure out if there was a conflicting plugin but disabling everything else doesn't seem to help.

The plugin recognizes all other chat-based triggers, but not "Floor X" to reset the sight/intuition state.

**Comment by syldrathecat** [comment]
If I write this as a test script, it shows the logic for creating the floor change regex is seemingly not functioning anymore:

```cs
uint id = 7270;
var str = Svc.Data.GetExcelSheet<LogMessage>()?.GetRow(id)?.Text?.ToString() ?? "Unknown";
var str2 = new Regex("^" + str.Replace("\u0002 \u0003\ufffd\u0002\u0003", @"(\d+)") + "$");
PluginLog.Information($"LocalString({id}) = '{str}'");
PluginLog.Information($"FloorChanged regex = '{str2}'");
```

Output:
```
03:26:11.382 | INF [ECommons] [Splatoon] Enabling script TestScript
03:26:11.382 | INF [ECommons] [Splatoon] LocalString(7270) = 'Floor '
03:26:11.382 | INF [ECommons] [Splatoon] FloorChanged = '^Floor $'
```

**Comment by Warchaild** [comment]
Can confirm, I am also having this issue in PotD

**Comment by carvelli** [comment]
Fixed as of v4.5. Not sure why this broke; it doesn't seem like the underlying LogMessage sheet has changed in any notable way.
